finalPrep/modZeroHero.js

```diff
@@ -1,41 +1,66 @@
 // Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 // Declare two variables - heroName AND specialAbility - set to strings
-
+var heroName =  'BetterGuy';
+var specialAbility = 'add egg wash on your baked goods for better browning';
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
-
+var greeting = 'Ho there! I am ' + heroName +'!';
+var catchphrase = `Are you mad? Putting those in the oven like that? Stop right now so I can ${specialAbility}!`;
 // Declare two variables - power AND energy - set to integers
-
+var power = 9000;
+var energy = 100;
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
 //   fullEnergy should add 150 to your current energy
-
+var fullPower = power * 500;
+var fullEnergy = energy + 150;
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
-
+var isHuman = true;
+var identityConcealed = true;
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
-
+var archEnemies = ['Limp Toast man', 'Eggs are Evil Boy', 'Soggy Biscuit Samuel']
+var sidekicks = ['Butterman', 'The Whipped Cream Witch', 'Gelato Gal']
 // Print the first sidekick to your console
-
+console.log(sidekicks[0]);
 // Print the last archEnemy to the console
-
+console.log(archEnemies[2]);
 // Write some code to add a new archEnemy to the archEnemies array
-
+archEnemies.push('The Wicked Waffle');
 // Print the archEnemies array to console to ensure you added a new archEnemy
-
+console.log(archEnemies)
 // Remove the first sidekick from the sidekicks array
-
+sidekicks.shift();
 // Print the sidekicks array to console to ensure you added a new sidekick
-
+console.log(sidekicks);
 // Create a function called assessSituation that takes three arguments - dangerLevel, saveTheDay, badExcuse
 //   - dangerLevel should be an integer
-//   - saveTheDay should be a string a hero would say once they save the day 
+//   - saveTheDay should be a string a hero would say once they save the day
 //   - badExcuse should be a string a hero would say if they are too afraid of the dangerLevel
-
+var dangerLevel
+var saveTheDay = 'Maillard smiles upon us today!'
+var badExcuse = 'YOUR DOUGH IS TOO WET, FOOL! This will never work!'
+
+// there is no need for the two other arguments, but it could be written as:
```

Comment:
Why do you think there isn't a need for two other arguments?  The instructions list out that the function should accept **three parameters**.